winml/test/common/runtimeParameters.h

```diff
@@ -0,0 +1,11 @@
+//-----------------------------------------------------------------------------
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//
+//-----------------------------------------------------------------------------
```

Comment:
I don't think this is the proper header. which reminds me, we need them in all the test source files. refer to any of the non-test source files, it should be:
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

Reply:
it's fine if you want to do this in a separate PR, but we do need to do this.

---
In reply to: [350506995](https://github.com/microsoft/onnxruntime/pull/2479#discussion_r350506995) [](ancestors = 350506995)